airbyte-integrations/connectors/source-shopify/source_shopify/spec.json

```diff
@@ -11,7 +11,8 @@
         "type": "string",
         "title": "Shopify Store",
         "description": "The name of your Shopify store found in the URL. For example, if your URL was https://NAME.myshopify.com, then the name would be 'NAME'.",
-        "pattern": "^(?!https:\/\/)(?!http:\/\/)(?!.*\\.myshopify\\.)(?!.*\\.myshopify).*",
+        "pattern": "^(?!https:\/\/)(?!http:\/\/).*",
+        "examples": ["my-store", "my-store.myshopify.com"],
```

Comment:
If we're going to update the examples, we should also update the `description` to say:

"The name of your Shopify store found in the URL or the shop URL itself. For example, if your shop is located at https://NAME.myshopify.com, then shop can be set to either 'NAME' or 'NAME.myshopify.com'."

Reply:
Done. Make sense!